### DISCUSSION TITLE
Can't get map to show up in rviz

### DISCUSSION THREAD

**Discussion by Illyvia** [iRobotEducation/create3_docs#610] [Questions]
### How are you connecting to your Create 3?

Wi-Fi and USB Ethernet (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Raspberry Pi 4/Ubuntu 22.04/Humble/VirtualBox ubuntu 22.04 VM/humble 

### Which version of ROS 2 is installed on your computer?

Humble

### Which firmware version is installed on your robot?

H.2.6

### Which RMW is your robot running?

FastRTPS

### Is the robot connected to a network? If so, what is the network type?

Home network

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

USB

### Describe your question.

I can't get the map topic to show up in rviz, everything is ntp synced, and I can access all topics on both the pi and the VM.
![image](https://github.com/user-attachments/assets/3404621e-0574-451a-bc5a-158b5e1be54b)

**Comment by alsora** [comment]
The error indicates that there's no tf message being published that includes the `map` reference frame.

How to get this will be very dependent on your system. Usually it's responsibility of the SLAM node to publish a tf from `map` to `odom` or `base_link`.

**Reply by Illyvia** [reply]
Oh ok, do you know of any fixes for this?
